hedera-dedupe-bigquery/pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <artifactId>hedera-bigquery-deduplication</artifactId>
```

Comment:
Ahh, missed that. Will do.

Additionally, am thinking about making this the pattern: hedera-<what_it_does>-<dest>
i.e. `hedera-deduplication-bigquery` and `hedera-etl-bigquery`.
More so because of  latter. Right now `hedera-etl-dataflow` is weird since that project is more general (apache beam) and can be run using other runners too. What do you say?

Reply:
This makes sense to me. Alternatively you could name it `hedera-etl-beam`.

Reply:
Done.
If we ever need to run a slightly modified version of pubsub→gcs pipeline, then we can create `hedera-etl-gcs`. That's why going for more descriptive then 'beam' suffix.